docs/developer/01-getting-started.md

```
@@ -0,0 +1,239 @@
+pip install cyanprintsdk
+```
+
+Or with Poetry:
+
+```bash
+poetry add cyanprintsdk
+```
+
+**Python Version**: Requires Python 3.12+
+
+### .NET / C#
```

markdownlint-cli2 (0.21.0)

[warning] No space inside hashes on closed atx style heading

(MD020, no-missing-space-closed-atx)